### ISSUE TITLE
Error with switching to a new library name

### ISSUE THREAD

**Issue by PrincePepper** [UAVCAN/gui_tool#52]
In the file `main.py` on  [line 505](https://github.com/UAVCAN/gui_tool/blob/develop/uavcan_gui_tool/main.py#L505), the `uavcan` library has not been changed to `pyuavcan_v0`.

Before:
```
InternalObjectDescriptor('uavcan', uavcan,
                                     'The main Pyuavcan module'),
```
Must be:
```
InternalObjectDescriptor('uavcan', pyuavcan_v0,
                                     'The main Pyuavcan module'),
```

**Comment by PrincePepper** [comment]
If you want, I can make a pull request with a change of one line :)

**Comment by pavel-kirienko** [comment]
Do it.

**Comment by coderkalyan** [comment]
Apologies for not getting to this, I'm currently traveling. @PrincePepper thanks for fixing this, I missed it while porting the gui_tool.

**Comment by pavel-kirienko** [comment]
Fixed in #53
